ad-aws-login.sh

```diff
@@ -86,6 +86,15 @@ if ! cat "${AWS_CONFIG}" | grep -q "^\[profile ${PROFILE_NAME}\]$"; then
     exit 2
 fi
 
+PROFILE_CONFIG=$(sed -n "/${PROFILE_NAME}/,/^ *$/p" ${AWS_CONFIG})
+
+if [[ -z $APP_NAME ]]; then
+    APP_NAME=$(echo ${PROFILE_CONFIG} |  sed 's/ *= */=/g' | tr ' ' '\n' | grep 'app=' | sed 's/^.*=//')
```

Comment:
ROLE_ARN=$(echo "${PROFILE_CONFIG}" | grep '^app *' | sed -E 's/role_arn *= *([^ ]*)$/\1/')

Reply:
This was if role_arn is not found, variable stays empty and role_arn gets selected by browser javascript